components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -10,6 +10,7 @@
 #include <map>
 #include <memory>
 #include <string>
+#include <utility>
```

Comment:
Is this for something in this PR or something else?

Reply:
this fixes linter error